docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,40 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+AES Re-Encryption Tool:
+-----------------------
+
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+=====================================================================
```

Comment:
I don't think it's actually a problem for rendering, but this underline is much, much longer than the title it underlines